Assets/MRTK/SDK/Features/Input/Handlers/ObjectManipulator.cs

```diff
@@ -153,17 +173,31 @@ public ReleaseBehaviorType ReleaseBehavior
             set => releaseBehavior = value;
         }
 
+        [FormerlySerializedAs("smoothingActive")]
         [SerializeField]
-        [Tooltip("Check to enable frame-rate independent smoothing.")]
-        private bool smoothingActive = true;
+        [Tooltip("Frame-rate independent smoothing for far interactions.")]
+        private bool smoothingFar = true;
 
         /// <summary>
-        /// Check to enable frame-rate independent smoothing.
+        /// Whether to enable frame-rate independent smoothing for far interactions.
         /// </summary>
-        public bool SmoothingActive
+        public bool SmoothingFar
         {
-            get => smoothingActive;
-            set => smoothingActive = value;
+            get => smoothingFar;
+            set => smoothingFar = value;
+        }
```

Comment:
Renaming a public property is a breaking change that would need to be documented in the release notes.

It would also be useful to keep the public `SmoothingActive` included, mark it as Obsolete, and point to  `SmoothingFar`.

Reply:
Thank you for the notes @CDiaz-MS. I recently pushed a new commit that reintroduced the SmoothingActive property and marked it obsolete with a note about the new properties. I also updated ObjectManipulator documentation with lots of detail about the new properties and a note about the deprecated one.